roles/grafana/tasks/main.yml

```diff
@@ -17,12 +17,29 @@
     name: "{{ __grafana_packages + __grafana_packages_extra }}"
     state: present
 
-- name: Template Grafana configuration
+- name: Get package facts now that Grafana is installed
+  package_facts:
+    manager: "auto"
+
+- name: Get Grafana version number from installed packages
+  set_fact:
+    grafana_version: "{{ ansible_facts.packages.grafana[0].version }}"
+
+- name: Template Grafana v7 configuration
```

Comment:
```suggestion
- name: Template Grafana v8 and earlier configuration
```

Reply:
@richm just missed this, will add as a followup - the change makes sense to me.

Reply:
Yeah - sorry - I didn't notice that the PR had been merged . . .